tests/torch/test_compressed_graph.py

```diff
@@ -91,64 +92,17 @@ def get_basic_quantization_config_with_hw_config_type(hw_config_type, input_samp
     return config
 
 
-def sort_dot(path):
-    with open(path, 'r', encoding='utf8') as f:
-        content = f.readlines()
-    start_line = 'strict digraph  {\n'
-    end_line = '}\n'
-    content.remove(start_line)
-    content.remove(end_line)
-
-    def graph_key(line, offset):
-        key = line.split(' ')[0].replace('"', '')
-        if '->' in line:
-            key += line.split(' ')[3].replace('"', '')
-            return int(key) + offset
-        return int(key)
-
-    sorted_content = sorted(content, key=partial(graph_key, offset=len(content)))
-    with open(path, 'w', encoding='utf8') as f:
-        f.write(start_line)
-        f.writelines(sorted_content)
-        f.write(end_line)
-
-
 def check_graph(graph: PTNNCFGraph, path_to_dot, graph_dir, sort_dot_graph=True):
     # pylint:disable=protected-access
     nx_graph = graph.get_graph_for_structure_analysis()
-    check_nx_graph(nx_graph, path_to_dot, graph_dir, sort_dot_graph=sort_dot_graph)
+    check_nx_graph(nx_graph, path_to_dot, graph_dir, sort_dot_graph)
```

Comment:
This call should therefore reference the common `check_nx_graph` function. Provided you use `pathlib.Path` objects, it should be trivial for you to build a full path for the corresponding argument in the common `check_nx_graph`.

Reply:
I can't use directly common `check_nx_graph` here cause I need to build the full path to the dot from `path_to_dot, graph_dir`, which is done in local `check_nx_graph`

Reply:
Done